multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -21,6 +21,11 @@ coreos:
       enable: true
       command: start
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
```

Comment:
nit: use `RequiredBy/Before` in the `load-rkt-stage1.service` definition, which is already in an equivalent conditional block. Then you can just get rid of this block.